synapse_data_warehouse/database_roles/V2.47.1__synapse_aggregate_database_roles.sql

```diff
@@ -0,0 +1,43 @@
+USE DATABASE {{ database_name }}; --noqa: JJ01,PRS,TMP
+
+-- Create "admin" database role which will own the synapse_aggregate objects
+CREATE OR REPLACE DATABASE ROLE SYNAPSE_AGGREGATE_ALL_ADMIN;
+
+-- Grant ownership of this database role to the proxy admin
+GRANT OWNERSHIP
+    ON DATABASE ROLE SYNAPSE_AGGREGATE_ALL_ADMIN
+    TO ROLE {{ database_name }}_PROXY_ADMIN; --noqa: JJ01,PRS,TMP
```

Comment:
Just a dumb question. Don't we want to create `{{ database_name }}_PROXY_ADMIN` before granting roles to it?

Reply:
NVM, this role should be created in admin/roles.sql.